src/modules/add-site/components/blueprints.tsx

```diff
@@ -39,6 +40,109 @@ interface DataViewBlueprint extends Blueprint {
 	categories: string[];
 }
 
+interface BlueprintIssuesModalProps {
+	warnings: Array< { feature: string; reason: string } > | undefined;
+	validationError: string | undefined;
+	fileName: string;
+	isOpen: boolean;
+	onClose: () => void;
+}
+
+function BlueprintIssuesModal( {
+	warnings,
+	validationError,
+	fileName,
+	isOpen,
+	onClose,
+}: BlueprintIssuesModalProps ) {
+	const { __ } = useI18n();
+
+	if ( ! isOpen ) {
+		return null;
+	}
+
+	const hasWarnings = warnings && warnings.length > 0;
+	const hasError = !! validationError;
+
+	return (
+		<Modal title={ __( 'Blueprint Details' ) } onRequestClose={ onClose } size="medium">
+			<VStack spacing={ 4 }>
+				<Text className="font-medium text-gray-900">{ fileName }</Text>
+
+				{ hasError && (
+					<div className="max-h-64 overflow-y-auto">
+						<VStack spacing={ 3 }>
+							<div>
+								<HStack alignment="topLeft" spacing={ 2 }>
+									<Icon icon={ error } className="text-red-500 mt-1 flex-shrink-0" size={ 20 } />
+									<VStack spacing={ 1 }>
+										<Text weight={ 600 } className="text-base">
+											{ __( 'Validation Error' ) }
+										</Text>
+										<Text className="text-sm text-gray-700">{ validationError }</Text>
+									</VStack>
+								</HStack>
+							</div>
+						</VStack>
+					</div>
+				) }
+
+				{ hasWarnings && (
+					<>
+						<Text>
+							{ sprintf(
+								// translators: %d is the number of unsupported features
+								__(
+									'The following %d feature(s) are not supported in Studio and will be automatically removed:'
+								),
+								warnings.length
+							) }
+						</Text>
+
+						<div className="max-h-64 overflow-y-auto">
+							<VStack spacing={ 3 } className="divide-y divide-gray-200">
+								{ warnings.map( ( warningItem, index ) => (
+									<div key={ index } className={ index > 0 ? 'pt-3' : '' }>
+										<HStack alignment="topLeft" spacing={ 2 }>
+											<Icon
+												icon={ caution }
+												className="text-orange-500 mt-1 flex-shrink-0"
+												size={ 20 }
+											/>
+											<VStack spacing={ 1 }>
+												<Text weight={ 600 } className="text-base">
+													{ warningItem.feature }
+												</Text>
+												<Text className="text-sm text-gray-700">{ warningItem.reason }</Text>
+											</VStack>
+										</HStack>
+									</div>
+								) ) }
+							</VStack>
+						</div>
+					</>
+				) }
+
+				{ hasWarnings && ! hasError && (
+					<div className="pt-4 border-t border-gray-200">
+						<Text className="text-sm text-gray-600">
+							{ __(
+								'Your blueprint will still work, but these features will be skipped during site creation.'
+							) }
+						</Text>
+					</div>
+				) }
+
+				<HStack alignment="right">
+					<Button variant="primary" onClick={ onClose }>
+						{ __( 'Got it' ) }
+					</Button>
+				</HStack>
```

Comment:
I think it's non-blocking, when all 3 unsupported features are listed, the CTA gets partly pushed out of view, and it can be scrolled into view.

We could update the modal template to make this HStack fixed at the bottom and make only the rest of the content scrollable.

<img width="2024" height="1424" alt="Image" src="https://github.com/user-attachments/assets/63a0d661-3d2b-42b2-9d63-690b1cc041c6" />